nixos/modules/hardware/rwedid.nix

```diff
@@ -0,0 +1,25 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.rwedid;
+in {
+  meta.maintainers = with lib.maintainers; [ janik ];
+
+  options = {
+    programs.rwedid = {
+      enable = lib.mkEnableOption (lib.mdDoc "Read and write EDID data over an I2C bus");
+      package = lib.mkPackageOptionMD pkgs "rwedid" { };
```

Comment:
We don't usually include a `.package` option in NixOS modules, unless there are multiple options available.